common/src/main/java/com/wynntils/models/gear/tooltip/GearTooltipFooter.java

```diff
@@ -74,7 +74,8 @@ public static List<Component> buildTooltip(GearInfo gearInfo, GearInstance gearI
         footer.add(tier);
 
         // restrictions (untradable, quest item)
-        if (gearInfo.metaInfo().restrictions() != GearRestrictions.NONE) {
+        // Note: "Zeer ID Test" item has null as restrictions...
```

Comment:
I think we should fix this when reading the API json instead.

Reply:
Can you look into that? I will keep this open as a reminder..

Reply:
Okay

Reply:
Actually, that was good luck I did! It did not have a null restriction. It had "Soulbound", which we could not parse and thus replaced with null. Presumably more items will have that in the future.